package.json

```diff
@@ -87,7 +87,7 @@
     "coverage": "yarn run jest --coverage",
     "format": "prettier --write .",
     "lint": "yarn lint:ci .",
-    "lint:ci": "eslint --config .eslintrc.js --ext .ts --ext .js",
```

Comment:
Using the `--config` option overrides the normal cascading behavior of eslintrc files.